### ISSUE TITLE
Map the network of prize-o-tron users

### ISSUE THREAD

**Issue by jimthedev** [AngularMN/prize-o-tron#14]
Prize-o-tron can be used by anyone, but it is nice to know which meetups are using it. Find a way to map this network of users.

Questions:
- Automatically track meetups using prize-o-tron vs allow opt in?
- If storing data outside of the repo in a database perhaps just prompt the prize-o-tron user to see if they want to notify us that they're using it.

**Comment by dalelotts** [comment]
Interesting idea. I was actually thinking of adding a database - we could capture this info as part of normal use  - we could also keep track of past winners, no shows, etc. We might also allow a meetups to specify a "no win period" for previous winners.

**Comment by jimthedev** [comment]
@dalelotts Good idea. I see you opened #15 for the no win period. I like it. Reading back I realized that this issue is poorly described so it has been updated to reflect what we're really trying to accomplish w/this issue.
